frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTabStatus.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Alert, AlertActionLink, Button, Skeleton } from '@patternfly/react-core';
+import {
+  LOG_REFRESH_RATE,
+  LOG_TAIL_LINES,
+} from '~/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/const';
+
+type LogsTabStatusProps = {
+  error?: Error;
+  loaded: boolean;
+  refresh: () => void;
+  onDownload: () => void;
+  canDownload: boolean;
+};
+
+const LogsTabStatus: React.FC<LogsTabStatusProps> = ({
+  error,
+  loaded,
+  refresh,
+  onDownload,
+  canDownload,
+}) => {
+  if (error) {
+    return (
+      <Alert
+        isInline
+        variant="danger"
+        title="An error occurred while retrieving the requested logs"
+        actionLinks={<AlertActionLink onClick={refresh}>Retry</AlertActionLink>}
+      >
+        <p>{error.message}</p>
```

Comment:
The error screen looks great 🥳 
<img width="707" alt="Screenshot 2023-08-31 at 11 25 36 AM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/c3225fec-5863-4635-8180-e32d8aeb38c1">

Future scope: This may be related to having better error message efforts. Can we have more specific error messages? For example for skipped nodes and old pipelines(we can specify why we can't get a pod for old pipeline).

Reply:
+1 to @christianvogt's comment here.

I have mentioned this case [in bullet #1 of my comment below](https://github.com/opendatahub-io/odh-dashboard/pull/1724#issuecomment-1700660641) and provided a screen for how it may look like. I was also thinking about hiding the toolbar in this case. Because all of the actions there are disabled in this case.

Reply:
https://github.com/opendatahub-io/odh-dashboard/issues/1734

Not part of the first pass.